airflow/models/dagrun.py

```diff
@@ -140,7 +141,7 @@ class DagRun(Base, LoggingMixin):
     triggered_by = Column(
         Enum(DagRunTriggeredByType, native_enum=False, length=50)
     )  # Airflow component that triggered the run.
-    conf = Column(PickleType)
+    conf = Column(JSONField(json=json), nullable=True)
```

Comment:
We should probably use JSONB for postgres cc @kaxil

Reply:
I noticed the backfill model was implemented using JSONField. But, I will update